src/createAction.js

```diff
@@ -29,7 +29,7 @@ export default function createAction(type, payloadCreator, metaCreator) {
     return action;
   };
 
-  actionHandler.toString = () => type;
+  actionHandler.toString = () => type.toString();
```

Comment:
So the question is if we are going to drop the Symbols support in a future version like v1.0.0 to stick with Redux best practices.